frontend/src/components/course/components/my-students-container/my-students-container.tsx

```diff
@@ -10,7 +10,12 @@ type Props = {
 
 const MyStudentsContainer: FC<Props> = ({ mentees }) => {
   if (!mentees.length) {
-    return <p className={styles.placeholder}>There are no students yet...</p>;
+    return (
+      <>
+        <h2 className={styles.studentsTitle}>My Students</h2>
```

Comment:
```suggestion
```
we have dublucate of `<h2 className={styles.studentsTitle}>My Students</h2>`. avoid it